modules/serverless-rn-1-28-0.adoc

```diff
@@ -38,7 +38,9 @@
 +
 With {ServerlessProductName} 1.28, the {ServerlessOperatorName} allows injecting the environment variable `ENABLE_SECRET_INFORMER_FILTERING_BY_CERT_UID` for both `net-istio` and `net-kourier`.
 +
-To prevent problems when upgrading from {ServerlessProductName} 1.28 to some future version, users must annotate their secrets with `networking.internal.knative.dev/certificate-uid:some_cuid`.
+If you enable secret filtering, all your secrets need to be annotated with  `networking.internal.knative.dev/certificate-uid: "<id>"`. Otherwise Knative Serving does not detect them, which leads to failures. This includes both new and existing secrets.
++
+In one of the following {ServerlessProductName} releases, secret filtering will become enabled by default. To prevent failures, annotate your secrets in advance.
```

Comment:
@kcarmichael08 Thanks. I phrased this as `In a future` initially, but it caused some confusion, so we've made the phrasing more explicit.
`in advance` is more precise than `now`, because we don't require the readers to do the change _now_, they just need to do it _before upgrading to the release that introduces this change_. Hence, _in advance_.
Leaving as is.